### ISSUE TITLE
Accessibility: Plugin single links and buttons missing hover states

### ISSUE THREAD

**Issue by amberhinds** [WordPress/wordpress.org#263]
On the plugin single page ([Gutenberg plugin test page](https://test.wordpress.org/plugins/gutenberg/)), the buttons and links are all missing hover states.

On hover, text links should have the underline removed.

On hover, buttons need a design change that either contrasts with the default state or is other wise visually distinct (switching from solid to underline or adding an icon, for example).

**Comment by ryelle** [comment]
> the buttons and links are all missing hover states.

It looks like these are not using the correct button styles — but if the old `button` classes are replaced with the block classes `wp-block-button__link` in the markup, that should help.

> On hover, text links should have the underline removed.

Yeah, this is a bug.

> On hover, buttons need a design change that either contrasts with the default state or is other wise visually distinct (switching from solid to underline or adding an icon, for example).

We recently [had a discussion about button styles](https://github.com/WordPress/wporg-parent-2021/issues/132), and from my research and understanding of the guidelines, the fact that the mouse cursor changes from default to pointer is enough of a change — although by default, the blue buttons _should_ get slightly darker on hover. https://wordpress.org/download/ shows these buttons the way they're supposed to work.

**Comment by StevenDufresne** [comment]
@WordPress/meta-design Do these need to be buttons? Can they not just be links? I don't know if we have an agreed-upon scenario where a button is preferred.

| Before | After |
|--------|--------|
| <img width="411" alt="Screenshot 2024-04-22 at 10 22 52 AM" src="https://github.com/WordPress/wordpress.org/assets/1657336/880f3cbc-fe5a-43ce-b20c-4175cab20fb4"> | <img width="383" alt="Screenshot 2024-04-22 at 10 22 48 AM" src="https://github.com/WordPress/wordpress.org/assets/1657336/4ea7fb97-646d-420b-aa1f-a8a978e20214"> | 

## Edit
The ratings section above doesn't include a button.

<img width="425" alt="Screenshot 2024-04-22 at 10 25 59 AM" src="https://github.com/WordPress/wordpress.org/assets/1657336/4bfb5383-2286-45ed-94f1-bc48751c321a">

**Comment by jasmussen** [comment]
Links look good 👍 👍

**Comment by ndiego** [comment]
> Links look good 👍 👍

Agreed, links look good and removing the buttons also increases the prominence of the Download and Live Preview (for those plugins that have it) buttons.

**Comment by StevenDufresne** [comment]
I made underlined as the default for the theme and updated button states. If any components are missing links or inconsistent, let's open up new tickets for those specific changes.

Thanks!
